tests/test_adjoint_jacobian.py

```diff
@@ -739,15 +739,24 @@ def test_controlled_jacobian(self, par, n_qubits, control_value, operation, tol)
 
             def circuit(p):
                 qml.StatePrep(init_state, wires=range(n_qubits))
-                qml.RX(p[0], 0)
-                qml.ctrl(
-                    operation(p[1], wires=range(n_qubits - num_wires, n_qubits)),
-                    control_wires,
-                    control_values=[
-                        control_value or bool(i % 2) for i, _ in enumerate(control_wires)
-                    ],
-                )
-                qml.RY(p[2], 0)
+                if operation.num_params == 3:
```

Comment:
Nice spot. That is provision for testing adjoint on `C(Rot)`, because `Rot` has three parameters. This could not go in before [5198](https://github.com/PennyLaneAI/pennylane/pull/5198). Now that the last is merged, I should be able to restore the `C(Rot)` test.